package.json

```diff
@@ -10,23 +10,28 @@
     "test": "jest"
   },
   "devDependencies": {
-    "browserify": "11.0.1",
-    "expect.js": "0.3.0",
+    "browserify": "16.2.2",
```

Comment:
Might also be worth adding the `engines` property

Reply:
Added